### ISSUE TITLE
train, test, and validation datasets

### ISSUE THREAD

**Issue by junxnone** [junxnone/junxnone.github.io#204]
# train dataset
是一些我们已经知道输入和输出的数据集训练机器去学习，通过拟合去寻找模型的初始参数。例如在神经网络（Neural Networks)中， 我们用训练数据集和反向传播算法（Backpropagation）去每个神经元找到最优的比重（Weights)。

# validation dataset
也是一些我们已经知道输入和输出的数据集，通过让机器学习去优化调整模型的参数，在神经网络中， 我们用验证数据集去寻找最优的网络深度（number of hidden layers)，或者决定反向传播算法的停止点；在普通的机器学习中常用的交叉验证（Cross Validation) 就是把训练数据集本身再细分成不同的验证数据集去训练模型。

# test dataset

用户测试模型表现的数据集，根据误差（一般为预测输出与实际输出的不同）来判断一个模型的好坏。

# Reference
- [ ] [测试数据集（Test Set) 和验证数据集 (Validation Set) 之间的区别](https://zhuanlan.zhihu.com/p/29133576)
